packages/ui-components/src/icon/widgets/tabbarsvg.ts

```diff
@@ -54,6 +62,13 @@ export namespace TabBarSvg {
   }
 
   export const defaultRenderer = new Renderer();
+
+  export interface IOptions extends TabBar.IOptions<any> {
```

Comment:
Let's keep the templating

```suggestion
  export interface IOptions<T> extends TabBar.IOptions<T> {
```

Reply:
thanks it was giving me errors, so I removed it, adding it to `IOptions` fixes that